packages/lit-dev-content/site/_includes/footer.html

```diff
@@ -50,6 +50,19 @@
         </svg>
       </a>
 
+      <a
+        href="https://goo.gle/Lit"
```

Comment:
I thought goo.gle was kinda deprecated? https://developers.googleblog.com/2018/03/transitioning-google-url-shortener.html

Why not a direct link, it will load faster?

Reply:
goo.gl is deprecated goo.gle is an internal-only vanity link

Reply:
We also use `lit.dev/slack-invite` should we just make a `lit.dev/youtube`?

Reply:
Ahhh ok. TIL!

Reply:
Oh yeah, that's a good idea! It's easier to remember IMO than the goo.gle one.

+1